### ISSUE TITLE
Accessibility: 3 dot menus read as "unlabeled" with talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [hjiangsu/thunder#1704]
### Bug Description

I just noticed that the three dot menus now read as unlabeled when using the TalkBack screen reader.

### Expected Behaviour

The 3. menus should be labeled more options.

### Steps to Reproduce

1. Turn on TalkBack from Android System Settings.
2. Open thunder
3. Tap three dot buttons

### Additional Context

![Image](https://github.com/user-attachments/assets/9b81e8bf-f54e-44ca-8034-517c8cafe54e)
![Image](https://github.com/user-attachments/assets/18afb76c-799f-469a-a9ef-249a0ab0dc2e)
![Image](https://github.com/user-attachments/assets/6315d7e4-1c7d-40d5-9f5a-e48184d7416d)

### App Version

0.6.1

### Device

OnePlus Nord N200

### OS

Android 14
